gr-qtgui/include/gnuradio/qtgui/freq_sink_c.h

```diff
@@ -128,6 +128,7 @@ namespace gr {
       virtual void set_update_time(double t) = 0;
 
       virtual void set_title(const std::string &title) = 0;
+      virtual void set_y_label(const std::string &label, const std::string &unit) = 0;
```

Comment:
Woops! It should be reflected in my branch now. I tested both block types in GRC.